data/syntax-highlighting/vim/syntax/meson.vim

```diff
@@ -75,42 +75,38 @@ syn keyword mesonBuiltin
   \ assert
   \ benchmark
   \ both_libraries
-  \ build_machine
   \ build_target
   \ configuration_data
   \ configure_file
   \ custom_target
+  \ debug
   \ declare_dependency
   \ dependency
   \ disabler
   \ environment
   \ error
   \ executable
   \ files
-  \ find_library
   \ find_program
   \ generator
   \ get_option
   \ get_variable
-  \ gettext
-  \ host_machine
   \ import
   \ include_directories
   \ install_data
+  \ install_emptydir
   \ install_headers
   \ install_man
   \ install_subdir
   \ install_symlink
-  \ install_emptydir
   \ is_disabler
   \ is_variable
   \ jar
   \ join_paths
   \ library
-  \ meson
   \ message
-  \ option
```

Comment:
Automatically generating this list using the output of the refman has led you to make a mistake.

Reply:
Thanks for pointing that out, it seems like I missed it because it's not defined in the yaml files? I can try adding a new file for it if that makes sense.

Should it go in `docs/yaml/functions/` ?

Reply:
It's not a meson.build function, but it is used in meson_options.txt which uses the same filetype.